server/tserver/src/main/java/org/apache/accumulo/tserver/log/LogSorter.java

```diff
@@ -242,7 +244,9 @@ public void startWatchingForRecoveryLogs(ThreadPoolExecutor distWorkQThreadPool)
         RecoveryStatus status = new RecoveryStatus();
         status.name = entries.getKey();
         try {
-          status.progress = entries.getValue().getBytesCopied() / (0.0 + conf.getAsBytes(Property.TSERV_WALOG_MAX_SIZE));
+          double progress = entries.getValue().getBytesCopied() / walBlockSize;
+          // to be sure progress does not exceed 100%
+          status.progress = Math.min(progress, 99.0);
```

Comment:
I suggest 99.9 instead of 99.0. It allows more room to see a changing number that's not "stuck" at 99.0%, but avoids the misleading behavior of being stuck at 100% for a long time if we just used 100.0 here.

Reply:
@ctubbsii Thank you for moving this to ACCUMULO-3807